MAINTAINERS.md

```diff
@@ -70,6 +70,7 @@ If you are an owner of the organization, you can see an automated list [here](ht
 - **Vitor Henrique ([@vitorhcl](https://github.com/vitorhcl))**: [18 December 2023](https://github.com/tldr-pages/tldr/issues/11771) — present
 - **cyqsimon ([@cyqsimon](https://github.com/cyqsimon))**: [28 December 2023](https://github.com/tldr-pages/tldr/issues/11864) — present
 - **Jongwon Youn ([@korECM](https://github.com/korECM))**: [29 December 2023](https://github.com/tldr-pages/tldr/issues/11892) — present
+- **Alejandro Cervera ([@tricantivu](https://github.com/tricantivu))**: [3 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — present
```

Comment:
```suggestion
- **Alejandro Cervera ([@tricantivu](https://github.com/tricantivu))**: [4 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — present
```

Just a minor suggestion and this PR is GTG. The date we refer to in the `MAINTAINERS.md` file is the date on which the invitation was sent and not when the issue was created.

Reply:
Understood. Sorry for the late reply.